<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>303</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
font:16px;
}
QPushButton {
border:none;
border-radius:4px;
background:#00B49C;
color:#EEEEEE;
font:16px;
}
QPushButton::pressed {
background:#55CBBC;
}
QPushButton::!enabled {
background:#AAAAAA;
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_VideoCodec">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_AudioCodec">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_GOP">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Url">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>460</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>rtsp://wowzaec2demo.streamlock.net/vod/mp4:BigBuckBunny_115k.mov</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Connect">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_Resolution">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="Open_red">
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
  </action>
  <action name="Close_Red">
   <property name="text">
    <string>Close(&amp;C)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>